src/style.css

```diff
@@ -1,3 +1,56 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: black;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  line-height: 60px;
+  text-decoration: none;
+  display: block;
+  color: black;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  color: #00b0f0;
+}
+
+.nav__link.is-active {
+  color: #00b0f0;
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
```

Comment:
The `.nav__link.is-active::after` pseudo-element uses `position: absolute`, but its parent element `.nav__link` does not have `position: relative`. This can cause positioning issues. Consider adding `position: relative` to `.nav__link` to ensure the pseudo-element is positioned correctly relative to its parent.